### PR TITLE
Add variable rsd_postgres_authenticator_password

### LINKED ISSUE #56: Use different values for POSTGRES_PASSWORD and  POSTGRES_AUTHENTICATOR_PASSWORD
Currently the role variable `{{ rsd_postgres_password }}` is used for both  env vars `POSTGRES_PASSWORD` and  `POSTGRES_AUTHENTICATOR_PASSWORD`.

See https://postgrest.org/en/stable/auth.html?highlight=authenticator#authentication-sequence for more details about the authenticator role.

### PR DESCRIPTION
**Pull request by Normo**
* closes #56

### CODE REVIEW

**Review by ruester — APPROVED** [review]

File: defaults/main.yml [hunk 1]
```diff
@@ -48,6 +48,7 @@ rsd_max_requests_github: "6"
 rsd_max_requests_gitlab: "6"
 rsd_max_requests_doi: "6"
 rsd_postgres_password: "changeme"
+rsd_postgres_authenticator_password: "ChangeMe"
```
Comment (tobiashuste) [inline defaults/main.yml:51]: I wonder if we should put it this way to not introduce breaking behavior? If people are not careful enough looking at a minor release, they might end up with a variable set to an insecure default.

```suggestion
rsd_postgres_authenticator_password: "{{ rsd_postgres_password }}"
```

We could keep the README as is to motivate setting the variable.

**Review by tobiashuste — COMMENTED** [review]
I made a suggestion to discuss about.

File: defaults/main.yml (hunk 1 above)
Comment (tobiashuste) [reply defaults/main.yml]: Thanks, you're of course right in this regard but thanks for changing!

**Review by tobiashuste — APPROVED** [review]
Thanks, fine from my side!

**Pull request merged**